packages/traefik/spec.yml

```diff
@@ -1,7 +1,7 @@
 ---
 name: traefik
 version: 1.0.0
-iteration: rc3
+iteration: 1
```

Comment:
good question - it looks like it does. On a fresh install, I end up with v1.0.0 and not rc3.